server/pbench/bin/index-pbench

```diff
@@ -15,6 +15,9 @@ from datetime import datetime, timedelta
 from collections import Counter
 from argparse import ArgumentParser
 
+import pprint
```

Comment:
Can we use `json.dump()` instead?

Reply:
Instead of pprint? If so, the plan is to get rid of pprint altogether. It's only used for debugging.